src/applications/find-va-forms/helpers/index.js

```diff
@@ -16,7 +16,7 @@ export const normalizeFormsForTable = forms =>
     // Derive the ID field.
     const idLabel = (
       <a href={downloadURL} rel="noopener noreferrer" target="_blank">
-        {id}
+        {id} {downloadURL.toLowerCase().includes('.pdf') && '(PDF)'}
```

Comment:
I used `includes` instead of `endsWith` because sometimes there is a query param at the end - `https://www.gsa.gov/cdnstatic/OF8-85.pdf?forceDownload=1`

Reply:
Not sure if it's desired, but we could perhaps do something like this if we know all the file extensions?

```jsx
// Check if the download URL has any extensions.
const downloadURLExtensions = downloadURL.toLowerCase().match(/(jpg|gif|doc|pdf)/)

// Derive the download URL extension.
const downloadURLExtension = downloadURLExtensions ? downloadURLExtensions[0] : '';

// ...

const idLabel = (
  <a href={downloadURL} rel="noopener noreferrer" target="_blank">
    {id} ({downloadURLExtension})
  </a>
);

// ...
```

Reply:
That is pretty cool but I think this is okay for now because the UX definitely expects a PDF. Man it would be disappointing to see a form float through as a `.doc` file lol.